scripts/docker/dockerfile_gcc-11

```diff
@@ -16,7 +16,7 @@ RUN git clone --recursive --branch $branch --single-branch --depth 1 https://git
 # Build/install TPLs via spack and then remove the temporary build directory on success
 RUN cd serac_repo && python3 ./scripts/uberenv/uberenv.py --spack-env-file=./scripts/spack/configs/docker/ubuntu20/spack.yaml \
                                                           --project-json=.uberenv_config.json \
-                                                          --spec=%gcc@11.1.0 --prefix=/home/serac/serac_tpls -k \
+                                                          --spec=%gcc@11.1.0~strumpack --prefix=/home/serac/serac_tpls -k \
```

Comment:
This doesn't work anymore either?

Reply:
no - it was the same weird problem. It showed up on my machine when I had docker installed - so I think it has something to do with that.